scipy/spatial/transform/_rotation.pyx

```diff
@@ -689,7 +689,7 @@ cdef class Rotation:
         quat = np.asarray(quat, dtype=float)
 
         if quat.ndim not in [1, 2] or quat.shape[len(quat.shape) - 1] != 4:
-            raise ValueError("Expected `quat` to have shape (4,) or (N x 4), "
+            raise ValueError("Expected `quat` to have shape (4,) or (N, 4), "
                              "got {}.".format(quat.shape))
```

Comment:
While we are at it.
```suggestion
            raise ValueError("Expected `quat` to have shape (4,) or (N, 4), "
                             f"got {quat.shape}.")
```

Did you check if there was a test for that validation?

Reply:
I wasn't sure about code style with respect to using f-strings. I will gladly replace `.format()`s with `f`s any day of the week :)